docs/src/preprocessing/preprocessing.md

```diff
@@ -117,90 +129,82 @@ The reconstructed surface will then look as in the following image.
 </figure>
 ```
 
-To construct the hierarchy for discrete piecewise-linear surfaces (1), we have to do the following.
+To construct the hierarchy for the discrete piecewise-linear example in (1), we have to do the following.
 
 (2):
-Each edge is distributed to the child whose box contains the edge's barycenter.
+Each edge is distributed to the child whose box contains the edge's barycenter (red dots in (2)).
 Splitting stops when the number of a box's edges slips below a
 threshold (usually $\approx 100$ faces in 3D, here: 6 edges).
 
 (3):
 For the closure, Jacobson et al. (2013) define *exterior vertices* (*exterior edges* in 3D)
-as boundary vertices of such a segmentation.
+as boundary vertices of such a segmentation (red dots in (3)).
 To find them, we traverse around each edge (face in 3D) in order, and
 increment or decrement for each vertex (edge) a specific counter.
 
 ```julia
-  # 2D
-  v1 = edge_vertices_ids[edge][1]
-  v2 = edge_vertices_ids[edge][2]
+v1 = edge_vertices_ids[edge][1]
+v2 = edge_vertices_ids[edge][2]
 
-  vertex_count[v1] += 1
-  vertex_count[v2] -= 1
-```
-```julia
-  # 3D
-  edge1 = face_edges_ids[face][1]
-
-  if edge_vertices_ids[edge1] == (v1, v2)
-      directed_edges[edge1] += 1
-  else
-      directed_edges[edge1] -= 1
-  end
+vertex_count[v1] += 1
+vertex_count[v2] -= 1
 ```
 
-In 2D, a vertex is declared as exterior if `vertex_count(vertex) != 0`.
-The exterior vertices are then connected to one arbitrary
-exterior vertex using appropriately oriented line segments
+In 2D, a vertex is declared as exterior if `vertex_count(vertex) != 0`, so there is not the same amount of edges in this box going into versus out of the vertex.
+To construct the closing surface, the exterior vertices are then connected to one arbitrary
+exterior vertex using appropriately oriented line segments:
 
 ```julia
-  edge = vertex_count[v] > 0 ? (closing_vertex, v) : (v, closing_vertex)
+edge = vertex_count[v] > 0 ? (closing_vertex, v) : (v, closing_vertex)
 ```
 
-In 3D, an edge is declared as exterior if `directed_edges[edge] != 0`
-The exterior edges are triangulated with one exterior vertex $k$ with orientation
-- ``\{i, j, k\}`` if `directed_edges[edge] < 0`
-- ``\{j, i, k\}`` if `directed_edges[edge] > 0`
+The resulting closed surface ``T.S \cup T.\overline{S}`` then has the same number of edges going into and out of each vertex.
 
 #### Incorrect evaluation
 
 If we follow the algorithm, we know that recursion stops if
 
-- The bounding box $T$ is a leaf
-- The query point $\mathbf{p}$ is outside the box
-
-(1): If $\mathbf{p}$ is outside the box, we calculate the winding number with the (red) closure of the box.
-
-(2): If $\mathbf{p}$ is inside the box, we use the (blue) edges distributed to the box.
-
-(3): In this case, it leads to an incorrect evaluation of the winding number.
+- the bounding box $T$ is a leaf or
+- the query point $\mathbf{p}$ is outside the box.
```

Comment:
```suggestion
- the query point ``\mathbf{p}`` is outside the box.
```
As mentioned before, please change all of these, as dollar signs are not officially supported.